### PR TITLE
PR to Commit the Last of Development Files

### PR DESCRIPTION
**Pull request by jiaccarino**
Included is the ci/cd updates for further merge to main
and removal of unnecessary files

### CODE REVIEW

**Review by matthewl121 — COMMENTED** [review]
Overall purpose of PR looks good.  Maybe we can add test-report.html and other test files we dont want to .gitignore.

File: .github/workflows/ec2backend.yml
Comment (matthewl121) [inline .github/workflows/ec2backend.yml:1]: Line 41: Should we also check if github.ref is also pushing to main?

Comment (jiaccarino) [reply .github/workflows/ec2backend.yml:1]: added it

**Review by matthewl121 — APPROVED** [review]
PR looks good to approve

**Review by glenscheng — APPROVED** [review]
LGTM!

**Pull request merged**